src/init/init-npm.js

```diff
@@ -73,7 +73,7 @@ export default async (dir, lintStyle) => {
 
     for (const profile of ['development', 'production']) {
       const envTarget = content.env[profile]['application/javascript'].presets.find(x => x[0] === 'env');
-      envTarget[1].targets.electron = electronPrebuilt.version;
+      envTarget[1].targets.electron = parseFloat(electronPrebuilt.version.split('.').slice(0, 2).join('.'));
```

Comment:
@yavorsky Afaik it only allows the `major.minor` syntax, not a patch version.  Floats can only have one decimal point 👍